bigtable-hbase-1.x-parent/bigtable-hbase-tools/pom.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>bigtable-hbase-1.x-parent</artifactId>
+    <groupId>com.google.cloud.bigtable</groupId>
+    <version>1.19.3-SNAPSHOT</version>
```

Comment:
please `x-version-update` comment so that release tooling can update this.

Also I just noticed that the package naming is off it should be `bigtable-hbase-tools-1x` can you update it here and in masteR?

Reply:
you mean 1.x?